### PR TITLE
feat: add informative tooltips to all fields

### PR DESCRIPTION
**Pull request by simboonlong**
feat: show alert tooltip when there is an endorsement awaiting holder's action
test: tooltip logic
chore: rename ownership to holdership
refactor: sec- out of ids

### CODE REVIEW

File: src/components/AssetInfo/TokenSideBar.test.tsx [hunk 1]
```diff
@@ -32,8 +32,8 @@ describe("tokenSideBar", () => {
     expect(wrapper.find("TokenSideBarHolder")).toHaveLength(1);
     expect(wrapper.find("TokenSideBarHolder section")).toHaveLength(3);
 
-    expect(wrapper.find("#sec-transferownership h4").text()).toStrictEqual("Transfer Holdership");
-    expect(wrapper.find("#sec-transferownership button").text()).toStrictEqual("Transfer");
+    expect(wrapper.find("#sec-transferholdership h4").text()).toStrictEqual("Transfer Holdership");
+    expect(wrapper.find("#sec-transferholdership button").text()).toStrictEqual("Transfer");
```
Comment (sumitnitsurat) [inline src/components/AssetInfo/TokenSideBar.test.tsx:36]: why this "sec-" in ID ?

File: src/components/AssetInfo/TokenSideBarTooltip.tsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+            <line x1="12" y1="17" x2="12.01" y2="17" />
+          </svg>
+          <ReactTooltip
+            id="tooltip-alert"
+            place="bottom"
+            type="dark"
+            effect="solid"
+            getContent={() => {
+              return <p>Endorsement awaiting</p>;
+            }}
```
Comment (sumitnitsurat) [inline src/components/AssetInfo/TokenSideBarTooltip.tsx]: getContent=(() => (<p> Endorsement awaiting</p>));

File: src/components/AssetInfo/TokenSideBarTooltip.tsx [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+import ReactTooltip from "react-tooltip";
+
+interface TokenSideBarTooltipProps {
+  id: string;
+  approvedBeneficiaryAddress?: string;
+}
+
+const TokenSideBarTooltip = ({ id, approvedBeneficiaryAddress }: TokenSideBarTooltipProps) => {
+  const getTooltipContent = () => {
+    switch (true) {
```
Comment (sumitnitsurat) [inline src/components/AssetInfo/TokenSideBarTooltip.tsx]: Try to refactor it as discussed.

**Comment by simboonlong** [comment]
abandoning this branch, will be redo-ing the feature at https://github.com/TradeTrust/tradetrust-website/pull/67

**Pull request closed without merging**